source/deploy/postgres-migration.rst

```diff
@@ -45,6 +48,7 @@ Before the migration
 -  Determine the migration window needed. This process requires you to stop the Mattermost Server during the migration.
 -  See the `schema-diffs <#schema-diffs>`__ section to ensure data compatibility between schemas.
 -  Prepare your PostgreSQL environment by creating a database and user. See the `database </install/prepare-mattermost-database.html>`__ documentation for details.
+-  If you are planning to run an iterative migration (running the pgloader several times), please see the :ref:`designated section<Iterative migrations>`.
```

Comment:
Has this link been verified as working?

Reply:
I literally pushed the PR for this :D There is a problem locally as Python causing an OOM :

![Screenshot 2023-12-06 at 15 34 59](https://github.com/mattermost/docs/assets/2153367/a6907509-3621-4310-8c43-1107ef9230b0)

Reply:
The Python issue is known. Docs doesn't have a fix for it yet. The workaround I use is to kill those processes.